src/main/java/org/jabref/logic/search/query/SearchQueryConversion.java

```diff
@@ -24,9 +23,9 @@ public static String flagsToSearchExpression(SearchQuery searchQuery) {
         return new SearchFlagsToExpressionVisitor(searchQuery.getSearchFlags()).visit(searchQuery.getContext());
     }
 
-    public static Query searchToLucene(SearchQuery searchQuery) {
+    public static String searchToLucene(SearchQuery searchQuery) {
```

Comment:
We loos strong typing, but I think, the conversion is easier now.

Reply:
Yes, because using the Analyzer here removes words like "a" and "the", which can lead to empty queries with `null`. This also made testing a bit confusing. For example, if the search term was `image`, I had to know its stem (`imag`) for the tests to pass.